src/core/lib/iomgr/resource_quota.cc

```diff
@@ -135,12 +138,32 @@ struct grpc_resource_quota {
 
   gpr_atm last_size;
 
+  /* Mutex to protect max_threads and num_threads */
+  /* Note: We could have used gpr_atm for max_threads and num_threads and avoid
+   * having this mutex; but in that case, each invocation of the function
+   * grpc_resource_user_alloc_threads() will have to do atleast two atomic loads
```

Comment:
grammar nit: s/will/would have had/, you are talking about a hypothetical. also, s/atleast/at least/

Reply:
Done